README.md

```diff
@@ -43,6 +43,8 @@ To protect this Lambda from other rouge Lambdas consuming all the available shar
 
 This value should be set to at least the number of Kinesis trigger shards.
 
+## Deployment
 
+This app is currently only deployed to PROD: there is no CODE stack. Also, [the Riff Raff configuration](./riff-raff.yaml) is only set to deploy the lambda updates: it doesn’t update the cloudformation.
```

Comment:
While we're here, this is something that tripped a few of us up recently looking at RiffRaff's history. Might be worth adding the `allowedStages` (I think that's what it's called) setting to the RiffRaff yaml file to only include PROD?

Reply:
Change made!